server/app/views/admin/programs/ProgramPredicateConfigureView.java

```diff
@@ -443,7 +443,7 @@ private DivTag renderQuestionHeaders(
                         .withData("question-id", String.valueOf(qd.getId())),
                     createScalarDropdown(qd, Optional.of(leafNode)),
                     createOperatorDropdown(qd, Optional.of(leafNode)))
-                .withClasses(COLUMN_WIDTH, iff(columnNumber++ != 1, "ml-16")));
+                .withClasses(COLUMN_WIDTH, "shrink-0", iff(columnNumber++ != 1, "ml-16")));
       }
     } else {
       int columnNumber = 1;
```

Comment:
suggestion (non-blocking):  It might be helpful to set `columnNumber` to `0` here like you did on line 289, just for consistency.